ofrak_core/ofrak/gui/server.py

```diff
@@ -1148,11 +1148,8 @@ async def set_projects_path(self, request: Request) -> Response:
     @exceptions_to_http(SerializedError)
     async def save_project_data(self, request: Request) -> Response:
         body = await request.json()
-        session_id = body["session_id"]
+        session_id = body["id"]
```

Comment:
This one in particular was not a bug, but the interchanging of `session_id` and `id` as request body elements introduced a bug somewhere else, so i standardized it in all requests. 

Regarding the `id` vs `session_id`, initially we had a `project_id` that would be unique to an individual project repository that would be the same every time you opened it. This is largely vestigial now and isn't used in the code. The `session_id` is a unique identifier unique to your server session so that you can have open multiple version of a project with the same `project_id`. I would have no problem removing the `project_id` and renaming `session_id` to just `id` but there could be use for the `project_id` in the future.

Reply:
Ok, I'm going to make a separate issue for this.

Reply:
> This one in particular was not a bug, but the interchanging of session_id and id as request body elements introduced a bug somewhere else, so i standardized it in all requests.

What was the actual bug? Does that merit updating the changelog (FIXED)? It would be helpful to know in this PR what bugs (if any) were fixed

Reply:
> Ok, I'm going to make a separate issue for this.

See #386.

Reply:
> > This one in particular was not a bug, but the interchanging of session_id and id as request body elements introduced a bug somewhere else, so i standardized it in all requests.
> 
> What was the actual bug? Does that merit updating the changelog (FIXED)? It would be helpful to know in this PR what bugs (if any) were fixed

sorry, not a bug but a mistake in another test elsewhere. There was a typo in `session_id` in `test_save_project_data` and `test_delete_from_project`.